Lib/test/test_importlib/resources/test_files.py

```diff
@@ -79,14 +79,14 @@ def test_module_resources(self):
         A module can have resources found adjacent to the module.
         """
         spec = {
-            'mod.py': '',
+            'mod_res.py': '',
             'res.txt': 'resources are the best',
         }
         _path.build(spec, self.site_dir)
-        import mod
+        import mod_res
 
-        actual = resources.files(mod).joinpath('res.txt').read_text(encoding='utf-8')
-        assert actual == spec['res.txt']
```

Comment:
How is this change pertinent to the reported issue? Because this code is shared with `importlib_resources` and tested by pytest, the bare assert provides better context and clearer syntax, so I'd prefer to keep it.

Reply:
Reverted in 39d135d15a14c47e34d254c325eaecb7c6214445